src/js/common/schemaform/validation.js

```diff
@@ -98,7 +98,7 @@ export function transformErrors(errors, uiSchema) {
  * should call addError to add the error.
  */
 
-export function uiSchemaValidate(errors, uiSchema, schema, formData, formContext, path = '') {
+export function uiSchemaValidate(errors, uiSchema, schema, pageData, formData, formContext, path = '') {
```

Comment:
Not even a little! But at this point, we probably should have more than just the schemaform documentation...Trouble is I don't really know what type `errors` is, what `formContext` is, nor what `path` is really used for, so... 😕